pkg/destroy/aws/aws.go

```diff
@@ -259,6 +261,12 @@ func (o *ClusterUninstaller) Run() error {
 		o.Logger.Debug(err)
 		return err
 	}
+
+	err = removeSharedTags(context.TODO(), tagClients, tagClientNames, o.Filters, o.Logger)
```

Comment:
hmm..
wouldn't the tagClients be modified to empty?

playing a little with this... _might have to read more_
https://play.golang.org/p/AX66D82CRVa

Reply:
> wouldn't the tagClients be modified to empty?

Nobody should be emptying `tagClients`, only `tagClientStack`.  This is what I was going for: https://play.golang.org/p/2uSCMbdb3DX

Reply:
hmm, we are filtering based on  if a resource was found or not.. https://github.com/openshift/installer/pull/2467/files/855a746b6794cb4b7d5c4b5c529860a6f49534c2#diff-ab65f0e2ba0237c5cc1429673c22cc08R205-R209

https://play.golang.org/p/-B-okX95o4F

Reply:
huh.  Pushed 855a746b6 -> df443dae5 using the [apparently][1] more reliable `append([]T(nil), a...)
` incantation (also from [the wiki][2]).

[1]: https://play.golang.org/p/fQ2-cfcoged
[2]: https://github.com/golang/go/wiki/SliceTricks#copy